docs/build/cmake_flags.md

```diff
@@ -0,0 +1,36 @@
+# CMake Flags
+
+## Build cache
+
+This project uses ccache automatically if it's available in the system. To disable this, use flag `-DAUTO_CCACHE=OFF`.
+
+## Building tests
+
+```sh
+cmake --preset <configure-preset> -DSC_BUILD_TESTS=ON
+cmake --build --preset <build-preset>
+```
+
+## Code formatting with CLangFormat
+
+To check code with CLangFormat run:
+```sh
+cmake --preset release-with-tests-conan -DSC_CLANG_FORMAT_CODE=ON
+cmake --build --preset release --target clangformat_check
```

Comment:
![image](https://github.com/user-attachments/assets/e9b6aae2-7bc5-4d6b-85a4-5721ed231206)

Reply:
Fixed

Reply:
CI says that all is well. Maybe you use old clang-format or Ubuntu 22.04